### PR TITLE
Menuicon

### PR DESCRIPTION
**Pull request by xihai01**
asha's home page design.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -12,10 +13,11 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
+    "react-scroll": "^1.8.4",
     "web-vitals": "^2.1.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```
Comment (simplyDonald) [inline client/package.json:20]: changed ports for client?

**Review by simplyDonald — CHANGES_REQUESTED** [review]
port change in packege JSON?

**Comment by xihai01** [comment]
@aalibarre

**Review by simplyDonald — APPROVED** [review]

**Review by simplyDonald — APPROVED** [review]

**Review by aalibarre — APPROVED** [review]

**Review by simplyDonald — APPROVED** [review]

**Review by xihai01 — COMMENTED** [review]
seems good to merge

**Pull request closed without merging**